src/test/java/org/openrewrite/kotlin/tree/AnonymousFunctionTest.java

```diff
@@ -23,6 +23,13 @@
 
 class AnonymousFunctionTest implements RewriteTest {
 
+//    @Override
+//    public void defaults(RecipeSpec spec) {
+//        spec
+//          .recipe(Recipe.noop())
+//          .allSources(spaceConscious());
+//    }
+
```

Comment:
This is how the assertion can be enabled for all test cases in a test class. But if individual test cases register their own after recipes, then this one won't be applied and would have to be chained manually.